proto/elys/epochs/v1/query.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+package elysnetwork.elys.epochs.v1;
+
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "elys/epochs/v1/genesis.proto";
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+
+option go_package = 	"github.com/elys-network/elys/x/epochs/types";
+
+// Query defines the gRPC querier service.
+service Query {
+  // EpochInfos provide running epochInfos
+  rpc EpochInfos(QueryEpochsInfoRequest) returns (QueryEpochsInfoResponse) {
+    option (google.api.http).get = "/evmos/epochs/v1/epochs";
```

Comment:
@AustinoBombino can we update the API endpoints as well so it uses the same base endpoint as the other modules endpoints see here 
![image](https://user-images.githubusercontent.com/121588426/223368285-ef0dd50a-3eaa-497b-8f02-12c5e1f51193.png)

thanks